crates/forge_loader/src/manifest.rs

```diff
@@ -120,6 +120,21 @@ pub struct AppInfo<'a> {
     pub id: &'a str,
 }
 
+#[derive(Default, Debug, Clone, PartialEq, Eq, Deserialize)]
+pub struct ModuleResolve<'a> {
+    pub function: Option<&'a str>
+}
+
+#[derive(Default, Debug, Clone, PartialEq, Eq, Deserialize)]
+pub struct Module<'a> {
+    #[serde(default, borrow)]
+    pub function: Option<&'a str>,
+    #[serde(default, borrow)]
+    pub resolver: Option<ModuleResolve<'a>>,
```

Comment:
Maybe we should just reuse `Resolver`, but wrap the `method` field in an `Option`.